hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -1528,7 +1528,7 @@
     {
       name: chip_sw_coremark
       uvm_test_seq: chip_sw_uart_tx_vseq
-      sw_images: ["//third_party/coremark/top_earlgrey:coremark_test:1"]
+      sw_images: ["//third_party/coremark/top_earlgrey:coremark_test:1:new_rules"]
```

Comment:
I had not seeing this refactor yet. Are we really creating something called "new_rules"? That is not a descriptive name, just a sort of timestamp, to be superseded by new_new_rules, etc.

Reply:
It's a temporary flag to indicate to the testbench (in `chip_env_cfg.sv`) to not apply the naming conventions of the current `opentitan_functest` rules.  Once everything is converted to the new rules, we'll go back and eliminate the flag as the older naming convention will no longer be relevant.

Reply:
I created #19973 so that we wont forget to eliminate `new_rules` in the dvsim configurations.